installvm.py

```diff
@@ -678,8 +680,8 @@ def copylog():
     vmInst.copyNetbootImage()
     vmInst.configGrub()
     vmInst.configDHCP()
-    vmInst.startInstallation()
-    vmInst.monitorInstallation()
+    #vmInst.startInstallation()
```

Comment:
@priyama2 can you please uncomment these, hope you tried code changes without this change